packages/service-worker/src/module.ts

```diff
@@ -115,10 +115,11 @@ export function ngswAppInitializer(
       }
     }
 
-    // Don't return anything to avoid blocking the application until the SW is registered or
-    // causing a crash if the SW registration fails.
+    // Don't return anything to avoid blocking the application until the SW is registered.
+    // Caught and ignore the error if SW registration fails.
```

Comment:
I think you mean "Catch and ignore" :+1:

Reply:
Thanks :-)